src/OpenLoco/Windows/PromptBrowseWindow.cpp

```diff
@@ -763,9 +763,10 @@ namespace OpenLoco::Ui::Windows::PromptBrowse
                         _newFiles.emplace_back(name, isDirectory);
                     }
                 }
-                catch (const fs::filesystem_error& err)
+                catch (const fs::filesystem_error& e)
                 {
-                    Console::error("Invalid directory or file: %s", err.what());
+                    Console::error("Invalid directory or file: %s", e.what());
+                    Ui::showMessageBox("Exception: ", e.what());
```

Comment:
Again don't think this needs an showMessageBox this is a window so clearly the normal window ui can be used.

Reply:
removed this